doc/en/user/source/extensions/wps-download/mapAnimationDownload.rst

```diff
@@ -225,3 +227,85 @@ Decoration Layout
 | The decorators draw on the image one after the other, so the order of the decorators in the layout file is important: the first decorator output will appear under the others.
 | Decorators are described in detail in the :ref:`wms_decorations` section.
 
+
+Sample variable-based DownloadAnimation request
++++++++++++++++++++++++++++++++++++++++++++++++
+
+If you want to animate a map based on variables, proceed as follows:
+
+* specify the ``animateParam`` variable. You can use this to animate SLD variables, as described in ``Variable substitution in SLD``
+* specify the ``animateValues`` parameter as a comma separated list of your values that will be used in the animation
+* omit the ``time`` parameter, as it cannot be used in conjunction with the ``animateParam`` parameter
+* optionally specify the ``fps`` parameter to define the frame per seconds of the output video (defaults to one)
+
+A sample animation request can look as follows:
+
+ .. code-block:: xml
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <wps:Execute version="1.0.0" service="WPS"
+                 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.opengis.net/wps/1.0.0"
+                 xmlns:wfs="http://www.opengis.net/wfs" xmlns:wps="http://www.opengis.net/wps/1.0.0"
+                 xmlns:ows="http://www.opengis.net/ows/1.1" xmlns:gml="http://www.opengis.net/gml"
+                 xmlns:ogc="http://www.opengis.net/ogc" xmlns:wcs="http://www.opengis.net/wcs/1.1.1"
+                 xmlns:xlink="http://www.w3.org/1999/xlink"
+                 xsi:schemaLocation="http://www.opengis.net/wps/1.0.0 http://schemas.opengis.net/wps/1.0.0/wpsAll.xsd">
+      <ows:Identifier>gs:DownloadAnimation</ows:Identifier>
+      <wps:DataInputs>
+        <wps:Input>
+          <ows:Identifier>bbox</ows:Identifier>
+          <wps:Data>
+            <wps:BoundingBoxData crs="EPSG:4326">
+              <ows:LowerCorner>-180 -90</ows:LowerCorner>
+              <ows:UpperCorner>180 90</ows:UpperCorner>
+            </wps:BoundingBoxData>
+          </wps:Data>
+        </wps:Input>
+        <wps:Input>
+          <ows:Identifier>animateParam</ows:Identifier>
+          <wps:Data>
+            <wps:LiteralData>env</wps:LiteralData>
```

Comment:
Seems like any parameter can be used, like in the animator. Which means, the documentation should state so, rather than referring to "SLD variable substitution" as the only possibility (besides time).

Reply:
Yes this is because i feel unsure if we shall support other parameters besides env. i haven't tested the behaviour with different layers and at least the bbox parameter would be conflict with the current process arguments. I think i will reduce this to only supporting env variables

Reply:
I think it's good to have some liberty, there are other variables that could be used for animations, like:
- elevation
- custom dimensions
- view params
I'd suggest to just document which parameter names might be problematic.

Reply:
i will have a look at supporting all parameters, but this may take some time

Reply:
It's just a matter of amending the documentation, mind, the code seems to be generic enough as it is.